composer.json

```diff
@@ -32,7 +32,8 @@
 		"symfony/yaml": "~2.0",
 		"symfony/finder": "~2.0",
 		"silverstripe/testsession": "*",
-		"silverstripe/framework": "^4.0.0"
+		"silverstripe/framework": "^4.0.0",
+		"chillu/fakedatabase": "*"
```

Comment:
If you look at my comment starting with "Given we already have a testsession module...", I recommended to add this dependency and its two custom classes to the `testsession` module instead. You can use `testsession` without `behat-extension`, and the FakeDatabase stuff can be useful without Behat